src/groups/bmq/bmqc/bmqc_orderedhashmapwithhistory.t.cpp

```diff
@@ -332,6 +332,37 @@ static void test6_eraseThenGc()
     obj.gc(now, BATCH_SIZE);
 }
 
+static void test7_gcThenInsert()
+{
+    // ------------------------------------------------------------------------
+    // HISTORY
+    //
+    // Concerns:
+    //   The same GUID gets inserted after 'erase' (as an undetected duplicate)
```

Comment:
Message GUID is from `bmqt` package. `bmqc` package doesn't depend on `bmqt`, so it's not good to mention it here. `bmqc_orderedhashmapwithhistory` is a general-purpose container, so it's better to rephrase the comment using general word "elements"

Secondly, the comment mentions "undetected duplicates". This instantly raises a question whether `bmqc::OrderedHashMapWithHistory` has a duplicate detection mechanism. In fact, it doesn't, and this comment refers to another layer of the broker logics, so it should not be mentioned here at all

```suggestion
    //   The same element gets inserted after 'erase'
```